boxes/constants.py

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
Probably `contants.py` is not the best name, since it is not clear that it is only used for tests.